site/docs/v1/tech/identity-providers/samlv2/azure-ad.adoc

```diff
@@ -0,0 +1,179 @@
+
+1. Creating a "dummy" X509 certificate and importing this into Key Master.
+2. Making a SAML Idp in FusionAuth (Entity Id) with a dummy certificate selected.
+3. Updating our AzureAD Enterprise application with values from our newly created Idp, thereby getting a valid X509 certificate.
+4. Revisiting our SAML Idp in FusionAuth and updating with the correct X509 certificate from the above step.
+5. Completing our integration.
+
+If you already have a valid X509 certificate, you can complete step two above using this valid certificate and skip a few of the steps below.
+
+With that discussed, let us begin!
+
+== Import a "Dummy" X509 Certificate
+
+The first step is to import a "dummy" certificate into Key Master. You can generate any X509 certificate as it will be changed later. Once generated, under Key Master (`Settings > Key Master`) click `Import Certificate`.
+
+image:identity-providers/azure-ad-saml/import-certificate[Import Certificate,width=1200,role=shadowed]
+
+Included is an example X509 certificate for sample use.
+
+```
+-----BEGIN CERTIFICATE-----
+MIIC1DCCAj2gAwIBAgIBADANBgkqhkiG9w0BAQ0FADCBhjELMAkGA1UEBhMCdXMx
+DzANBgNVBAgMBkRlbnZlcjETMBEGA1UECgwKRnVzaW9uQXV0aDEWMBQGA1UEAwwN
+RnVzaW9uQXV0aC5pbzETMBEGA1UECwwKRnVzaW9uQXV0aDEkMCIGCSqGSIb3DQEJ
+ARYVcmljaGFyZEBwaWVkcGlwZXIuY29tMB4XDTIyMDcxNjA1MDc1N1oXDTIzMDcx
+NjA1MDc1N1owgYYxCzAJBgNVBAYTAnVzMQ8wDQYDVQQIDAZEZW52ZXIxEzARBgNV
+BAoMCkZ1c2lvbkF1dGgxFjAUBgNVBAMMDUZ1c2lvbkF1dGguaW8xEzARBgNVBAsM
+CkZ1c2lvbkF1dGgxJDAiBgkqhkiG9w0BCQEWFXJpY2hhcmRAcGllZHBpcGVyLmNv
+bTCBnzANBgkqhkiG9w0BAQEFAAOBjQAwgYkCgYEA2FmZJHJFVEM9JwFd6Za87T0Z
+MtIL5djSFC/TBVqhCx15eauNGAV/RoulESA6qsI4LNrbJ8uEYDQa9UXAZCc9yRMa
+e/+E5XAApV4K06duo+vKon5L21YZ7HxzjUfL3bhLqKvpFWCQkNrH0rxgPCGwzh7N
+T24sFcKwaVvBdknm9i8CAwEAAaNQME4wHQYDVR0OBBYEFLdhYvqAwTBCEQsSZdKj
+UVR57CwDMB8GA1UdIwQYMBaAFLdhYvqAwTBCEQsSZdKjUVR57CwDMAwGA1UdEwQF
+MAMBAf8wDQYJKoZIhvcNAQENBQADgYEAIpQags/uHj0dyFcCtRcPeVEDUqBPZaGO
+M9kbFiGATlMNX4OiDvMUKs7Ag9109o0iLWPvLBQrMDn87fSy6+MUXZRS4asjzJCp
+5aVWSevI85H6xS8WXxFr21etaqfiE88Lw86gK5O4iKtMBtCnWA5iUc2EJ0citQ0G
+Pk8ybmMP1r8=
+-----END CERTIFICATE-----
+```
+
+== Create a SAML v2 Identity Provider
+
+Returning to the FusionAuth Admin UI, navigate to `Settings > Identity Providers > Add`.
+
+The identity provider we will be adding is a `SAML V2` identity provider.
+
+image:identity-providers/azure-ad-saml/idp-saml-configuration[IdP SAML configuration,width=1200,role=shadowed]
+
+The following fields will need to be completed.
+
+- _Name_. This is user-defined.
+- _Idp endpoint_. This value can be obtained from your AzureAD Application as demonstrated below.
+
+image:identity-providers/azure-ad-saml/step-four-idp-endpoint[Step Four,width=1200,role=shadowed]
+
+- _Verification key_. For this value, please use the dummy certificate imported above. We will change this later.
+
+Be sure to select `Debug Enabled`. This will ensure debug information is written to the `Event Log`. Remember to disable this option in production.
```

Comment:
```suggestion
Be sure to check `Debug Enabled`. This will ensure debug information is written to the Event Log, which you can see by navigating to [breadcrumb]#System -> Event Log#. 

Remember to disable this option when running in production.
```